Model/DirectoryElementCollection.cs

```diff
@@ -54,4 +74,219 @@ public DirectoryElement FindElementByItemName(string itemName)
             base.Add(item: item);
         }
     }
+
+    /// <summary>
+    /// Parses the given folder into DirectoryElements.
+    /// 
+    /// The previous collection of directory elements is cleared before.
+    /// The statusMethod to be passed optionally accepts a string
+    /// containing a short status text.
+    /// </summary>
+    /// <param name="folder">The folder to parse</param>
+    /// <param name="statusMethod">The method to call for status updates</param>
+    public void ParseFolderToDEs(string folder, Action<string> statusMethod)
+    {
+        Logger.Trace(message: $"Start Parsing Folder '{folder}'");
+        statusMethod("Scanning folder: Initializing ...");
+
+        if (_ExifTool == null)
+            throw new InvalidOperationException($"Cannot scan a folder (currently '{folder}') when the EXIF Tool was not set for the DirectoryElementCollection.");
+
+        if (folder.EndsWith(value: @"\"))
```

Comment:
Any particular reason for introducing this one? 
This actually seems to cause a problem. I have a bit of an obscure setup on my computer but the logic is that the app sits on `E:\Google Drive\Sajat\Jobs\Nem_Aktualis\Apps\GeoTagNinja\` - this folder (technically all of `E:\Google Drive\*.*`) doesn't entirely exist though, it's a `junction` based mirror of the `G:\` - when we further down call `string tmpStrParent = HelperStatic.FsoGetParent(path: folder);` it will try to get the parent of `E:` and return a very odd path instead, in particular the debug (!!!) subfolder of the app. So I can't navigate to `E:` 
When we keep the `\` [as in `E:\`] the app works okay.

Reply:
I actually just copy-pasted from the previous method HelperStatic.ExifGetExifFromFolder(string folderNameToUse). There, it read:
>         if (folderNameToUse.EndsWith(value: @"\"))
>         {
>             folderNameToUse = folderNameToUse.Substring(startIndex: 0, length: folderNameToUse.Length - 1);
>         }

I guessed that it should ensure that there is no path used like "E:\folderA\"... :)

That d... thing behaves differently - not nice:
![grafik](https://user-images.githubusercontent.com/1786902/213573399-5034150c-fb8b-4f82-9f3d-70f7ab6ac80b.png)

But looking at the result C:\Temp\ versus C:\Temp, we should be fine by just removing this removal:
![grafik](https://user-images.githubusercontent.com/1786902/213574063-673a7fb1-4cd3-4df9-bf36-6a046fdd01f8.png)